cavityDisplayGUI.py

```diff
@@ -96,13 +98,19 @@ def __init__(self, parent=None, args=None):
                     # .add_callback is called when statusPV changes value
                     statusPV.add_callback(partial(self.statusCallback, cavityWidget))
 
+                    # .add_callback is called when descriptionPV
```

Comment:
This comment seems half written haha

Reply:
You are correct! I absolutely started it and then got distracted lol